build-tools/packages/build-cli/src/commands/release/report.ts

```diff
@@ -386,6 +386,11 @@ export default class ReleaseReportCommand extends ReleaseReportBaseCommand<
 			char: "o",
 			description: "Output JSON report files to this directory.",
 		}),
+		baseFileName: Flags.boolean({
```

Comment:
Make this a string and then use it as the base filename. E.g. `--baseFileName foo` would yield `foo.caret.json`.